tests/views/test_session.py

```diff
@@ -1,30 +1,17 @@
 import base64
+from tests.views.test_view_base import ViewTestBase, drop_all_collections
+import json
 import unittest
 
-from mongoengine import disconnect
-
-from app import create_app
 from app.models.item import Item
 from app.models.user import User
 
 
-class SessionControllerTestCase(unittest.TestCase):
-
-    @classmethod
-    def setUpClass(cls):
-        disconnect(alias='default')
-        cls.app = create_app()
-        cls.app.config['TESTING'] = True
-        cls.client = cls.app.test_client()
-
-    @classmethod
-    def tearDownClass(cls):
-        disconnect(alias='default')
+class SessionViewTestCase(ViewTestBase):
 
     def setUp(self):
```

Comment:
You can replace everything from drop_all_collections() to self.token... with super().setUp()